src/demangle.cpp

```diff
@@ -15,11 +15,16 @@
 #include <string>
 #include <ctype.h>
 #include <assert.h>
+#include <stdexcept>
 
 #include "symutil.h"
 
 #ifdef _M_X64
 extern "C" void cvt80to64(void * in, long double * out);
+#elif _M_ARM64
+void cvt80to64(void * in, long double * out) {
+	throw std::runtime_error("cvt80to64 is only supported on x64 processors.");

```

Comment:
This produces a silent termination with msvc, so I'd prefer a printf + exit.

Otherwise looks good to me.

Reply:
Thanks. Have just implemented that. [CI succeeded again](https://github.com/dennisameling/cv2pdb/actions/runs/3080891346).

The resulting binary also starts correctly:

```
C:\repos\cv2pdb\bin\Release_ARM64>.\cv2pdb --help
Convert DMD CodeView/DWARF debug information to PDB files, Version 0.52
Copyright (c) 2009-2012 by Rainer Schuetze, All Rights Reserved

License for redistribution is given by the Artistic License 2.0
see file LICENSE for further details

usage: --help [-D<version>|-C|-n|-e|-s<C>|-p<embedded-pdb>] <exe-file> [new-exe-file] [pdb-file]
```